src/strings/PowerFxResources.en-US.resx

```diff
@@ -5139,7 +5139,7 @@
     <comment>Error Message.</comment>
   </data>
   <data name="WrnDelagationTableNotSupported" xml:space="preserve">
-    <value>Delegating this operation on table '{0}' is not supported.</value>
+    <value>This operation may not work as expected on large tables (>{0} rows). We are working on delegating formulas to support large data sets.</value>
```

Comment:
"We are working on delegating formulas to support large data sets" - wording here probably needs to be tweaked. 

What error does Power Apps show in this case?